utils/__snapshots__/message_html_to_component.test.jsx.snap

```diff
@@ -153,6 +153,14 @@ exports[`messageHtmlToComponent link with enabled a tooltip plugin 1`] = `
     target="_blank"
   >
     <LinkTooltip
+      attribs={
+        Object {
+          "class": "theme markdown__link",
+          "href": "http://www.dolor.com",
+          "rel": "noreferrer",
+          "target": "_blank",
+        }
```

Comment:
This snapshot doesn't match correctly: https://app.circleci.com/pipelines/github/mattermost/mattermost-webapp/17647/workflows/62bed7e2-0844-4f2e-a5c9-35fe04c97589/jobs/110150/steps